app/models/observation.rb

```diff
@@ -1,4 +1,8 @@
 class Observation < ActiveRecord::Base
   belongs_to :observer, class_name: "Dog"
   belongs_to :observed, class_name: "Dog"
+
+  scope :matches, ->(current_dog) { includes(:observed)
+                                    .where(observer_id: current_dog.id,
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 34) spaces for indenting an expression spanning multiple lines.